packages/core/locales/fr/zod.json

```diff
@@ -72,7 +73,8 @@
       "url": "lien",
       "uuid": "UUID",
       "cuid": "CUID",
-      "regex": "expression régulière"
+      "regex": "expression régulière",
+      "datetime": "datetime"
```

Comment:
@ismailajizou I would appreciate it if you could help me with the translation here and check line 22

Reply:
@ismailajizou Thank you! Merci!